SystemServices/SystemServices.cpp

```diff
@@ -67,7 +67,7 @@ using namespace std;
 
 #define API_VERSION_NUMBER_MAJOR 2
 #define API_VERSION_NUMBER_MINOR 2
-#define API_VERSION_NUMBER_PATCH 1
+#define API_VERSION_NUMBER_PATCH 2
```

Comment:
Pls update minor version since this is a new API added and reset patch version to 0. So (2.3.0). Also update the same in CHANGELOG.

Reply:
Done anand